src/content/structured/community/roadmap.mdx

```diff
@@ -28,22 +28,29 @@ Check out our [GitHub](https://github.com/orgs/mi6/projects/2/views/1) for a mor
 - Canary release of [date input component](/components/date-input).
 - Canary release of [date picker component](/components/date-picker).
 - Canary release of [multi-select component](/components/multi-select).
-- Development of [top navigation layout pattern](/patterns/navigation-patterns/top-navigation-layout).
-- Development of [side navigation layout pattern](/patterns/navigation-patterns/side-navigation-layout).
-- Development of [file upload component](/components/button/code/#file-upload).
-- Development of [toggle button component](/components/toggle-button).
-- Added [component images to component galley](/components).
+- Canary release of [tree view component](/components/tree-view).
+- Canary release of [horizontal card component](/components/horizontal-card).
+- Release of [toggle button component](/components/toggle-button).
+- Release of [accordion component](/components/accordion).
+- Improvements to code previews (StackBlitz, hide/show code, imports).
+- Refresh of website including homepage and developer and designer landing pages.
+- Improvements to testing (visual regression, Cypress integration).
+- Custom theme for AG grid component.
+- Token and theming of code components for dark mode.
```

Comment:
I think it would be fine to just put "components"? Not sure "code components" is very clear
```suggestion
- Tokens and theming of components for dark mode.
```